src/Models/ApplicantModel.php

```diff
@@ -557,13 +557,39 @@ public function getApplicantByEmail(string $email): array
         return $result;
     }
 
-    public function setAptitudeScore($id, $score): bool
+    public function getAptitudeScore(int $id): array
+    {
+        $sql = 'SELECT `aptitude`, `assessmentNotes` FROM applicants_additional WHERE `id` = :id;';
```

Comment:
`applicants_additional` needs backticks

Reply:
Resolved, this was the old method that we started to use and renamed! - can tell you didn't do the code review.